code/datums/mind.dm

```diff
@@ -1022,10 +1022,16 @@
 				var/new_total = input(usr, "Select a new value:", "Modify total blood") as null|num
 				if(isnull(new_total) || new_total < 0)
 					return
+
 				var/datum/antagonist/vampire/vamp = has_antag_datum(/datum/antagonist/vampire)
+				if(new_total < vamp.bloodtotal)
+					if(alert(usr, "Note that reducing the vampire's total blood may remove some active powers. Continue?", "Confirm New Total", "Yes", "No") == "No")
+						return
+					QDEL_LIST(vamp.powers)
+
```

Comment:
going to need to call `update_sight()` here or, make use of the `remove_ability()` proc.

Reply:
I just realized that I can't simply call `QDEL_LIST(powers)` because they linger in the mind's `spell_list`. I changed this instance and others to use `remove_ability()` instead.